extensions/helix/package.build.ts

```diff
@@ -274,6 +274,225 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+          },
+          "r": {
+            text: "Rename symbol",
+            command: "editor.action.rename",
+          },
+          "h": {
+            text: "Select symbol reference",
+            command: "editor.action.referenceSearch.trigger",
+          },
+        },
+
+        "window": {
+          title: "Window",
+          items: {
+            "w": {
+              text: "Goto next window",
+              command: "workbench.action.nextEditor",
+            },
+            "s": {
+              text: "Horizontal bottom split",
+              command: "workbench.action.splitEditorDown",
+            },
+            "v": {
+              text: "Vertical right split",
+              command: "workbench.action.splitEditor",
+            },
+            "t": {
+              text: "Transpose splits",
+              command: "workbench.action.toggleEditorGroupLayout",
+            },
+            // "f": {
+            //   text: "Open files in selection (hsplit)",
+            //   command: "dance.selections.open", function needs to be modified
+            // },
+            // "F": {
+            //   text: "Open files in selection (vsplit)",
+            //   command: "dance.selections.open", function needs to be modified
+            // },
+            "q": {
+              text: "Close window",
+              command: "workbench.action.closeActiveEditor",
+            },
+            "o": {
+              text: "Close all other windows (Current window only)",
+              command: "workbench.action.closeOtherEditors",
+            },
+            "h": {
+              text: "Jump to the split on the left",
+              command: "workbench.action.focusLeftGroup",
+            },
+            "j": {
+              text: "Jump to the split below",
+              command: "workbench.action.focusBelowGroup",
+            },
+            "k": {
+              text: "Jump to the split above",
+              command: "workbench.action.focusAboveGroup",
+            },
+            "l": {
+              text: "Jump to the split to the right",
+              command: "workbench.action.focusRightGroup",
+            },
+            "H": {
+              text: "Swap with the split to the left",
+              command: "workbench.action.moveActiveEditorGroupLeft",
+            },
+            "J": {
+              text: "Swap with the split below",
+              command: "workbench.action.moveActiveEditorGroupDown",
+            },
+            "K": {
+              text: "Swap with the split above",
+              command: "workbench.action.moveActiveEditorGroupUp",
+            },
+            "L": {
+              text: "Swap with the split to the right",
+              command: "workbench.action.moveActiveEditorGroupRight",
+            },
+            // "n": { Not easily possible. Neccessary?
+            //   text: "New split scratch buffer",
+            //   command: "",
+            // },
```

Comment:
No need to do it here, but perhaps we can check if the editor has horizontal / vertical splits already (maybe with `vscode.window.tabGroups`?), create them if needed (maybe `vscode.window.showTextDocument()` or using `View: Split Editor`), and then an empty file there? Also I'm not sure all of these steps are possible.